### ISSUE TITLE
SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for accessibility-db-mysql failed: Name...

### ISSUE THREAD

**Issue by jobara** [accessibility-in-action/platform#1686]
**`Illuminate\Database\QueryException`**

SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for accessibility-db-mysql failed: Name or service not known (SQL: select * from `sessions` where `id` = CUISn6nt0KUsXDNp6mK8b1t3SgTf5nJ8S31Q691e limit 1)

:dart: Illuminate\Database\Connection::runQueryCallback
:page_facing_up: /app/vendor/laravel/framework/src/Illuminate/Database/Connection.php

```php
758  // lot more helpful to the developer instead of just the database's errors.
759  catch (Exception $e) {
760*     throw new QueryException(
761          $query, $this->prepareBindings($bindings), $e
762      );
```
[Open in Flare](https://flareapp.io/errors/3524793/latest)

**Comment by jobara** [comment]
Has not re-occurred for a while.
